R/tidy_ard_order.R

```diff
@@ -0,0 +1,79 @@
+#' Standard Column Order of ARD
```

Comment:
I love this pair of functions! What do you think of documenting them in the same help file? I think🎵they belong together 🎵